doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -1131,7 +1131,7 @@ \subsubsection{Inputs}\label{inputs-3-026}
 
 \paragraph{Field: Design Heat Recovery Water Flow Rate}\label{field-design-heat-recovery-water-flow-rate-000}
 
-This is the design flow rate used if the heat recovery option is being simulated. If this value is greater than 0.0 then a heat recovery loop must be specified and attached to the chiller using the next 2 node fields. To determine how the heat recovery algorithm works look at the Engineering Manual at the Chiller:Electric with Heat Recovery section. The units are in cubic meters per second. This field is autosizable.~ When autosizing, the flow rate is simply the product of the design condenser flow rate and the condenser heat recovery relative capacity fraction set in the field below. Note that heat recovery is only available with Condenser Type = WaterCooled.
+This is the design flow rate used if the heat recovery option is being simulated. If this value is greater than 0.0 then a heat recovery loop must be specified and attached to the chiller using the next 2 node fields. To determine how the heat recovery algorithm works look at the Engineering Manual at the Chiller:Electric with Heat Recovery section. The units are in cubic meters per second. This field is autosizable.~ When autosizing, the flow rate is simply the product of the design condenser flow rate and the condenser heat recovery relative capacity fraction set in the field below.
```

Comment:
@rraustad it looks like this comment "Note that heat recovery is only available with Condenser Type = WaterCooled" that you deleted from `Chiller:Electric` documentation is still lingering in the documentation for the same input field for `Chiller:Electric:EIR` on [Line 1704](https://github.com/NREL/EnergyPlus/blob/7636e6b3e972dad33c41815304938b2139dde367/doc/input-output-reference/src/overview/group-plant-equipment.tex#L1704C2-L1704C2). Would be good to remove it there as well to avoid confusing users.